docs/doxygen-user/volatility_dsp.dox

```diff
@@ -0,0 +1,32 @@
+/*! \page volatility_dsp_page Volatility Data Source Processor
+
+\section Overview
+
+The Volatility data source processor runs Volatility on a memory image and saves the individual module results. If the disk image associated with the memory image is also available, it will create Intesting Item artifacts linking the Volatility results to files in the disk image.
+
```

Comment:
Intesting => Interesting

Might also want to say "individual Volatility module results" just to be clear we are talking about their modules, not ours.